solutions/observability/apm/apm-server-setup.md

```diff
@@ -0,0 +1,88 @@
+---
+navigation_title: Set up
+applies_to:
+  stack: ga
+  serverless: unavailable
+---
+
+# Set up APM Server
+
+:::{tip}
+Just getting started?
+The fastest way to get started with Elastic APM is with Elastic Cloud Serverless.
+Read more in [](/solutions/observability/apm/get-started.md).
+:::
+
+APM Server receives performance data from your APM agents, validates and processes it, and then transforms the data into {{es}} documents. If you’re on this page, then you’ve chosen to self-manage the Elastic Stack, and you now must decide how to run and configure the APM Server. There are two options, and the components required are different for each:
```

Comment:
There's probably an opportunity to be more OTel-inclusive with our phrasing here. @theletterf should we?

```suggestion
APM Server receives performance data from your APM agents and Elastic Distribution of OpenTelemetry (EDOT) SDKs, validates and processes it, and then transforms the data into {{es}} documents. If you’re on this page, then you’ve chosen to self-manage the Elastic Stack, and you now must decide how to run and configure the APM Server. There are two options, and the components required are different for each:
```